app/src/main/res/layout/unfriendly_activity.xml

```diff
@@ -107,5 +107,28 @@
                 android:layout_height="46dp"
                 android:layout_width="46dp" />
         </LinearLayout>
+
+        <LinearLayout
```

Comment:
Mind including a screenshot in the PR of what this update looks like?

Reply:
No problem, I'll add it in the description.